src/tools/cephfs/cephfs-shell

```diff
@@ -718,6 +717,22 @@ class CephFSShell(Cmd):
         """
         Copy a file/directory from CephFS to given path.
         """
+        if not is_file_exists(args.remote_path):
+            if not is_dir_exists(args.remote_path):
+                set_exit_code_msg(errno.ENOENT, "error: no file/directory"
+                                                " found at specified remote "
+                                                "path")
+                return
+        if os.path.isfile(args.local_path) and not args.force:
```

Comment:
this and the block below can be under one `if` block.

Reply:
Okay, I'll push the required change.

Reply:
Done